azure/functions/decorators/function_app.py

```diff
@@ -3276,6 +3265,7 @@ def __init__(self, auth_level: Union[AuthLevel, str], *args, **kwargs):
         DecoratorApi.__init__(self, *args, **kwargs)
         HttpFunctionsAuthLevelMixin.__init__(self, auth_level, *args, **kwargs)
         self._require_auth_level: Optional[bool] = None
+        self.functions_bindings: Optional[Dict[Any, Any]] = None
 
     def get_functions(self) -> List[Function]:
```

Comment:
We need to make sure that all the places where this is called in the worker have exception handling setup correctly. Otherwise, this will throw an error and without a catch, python will exit with code 1

Reply:
The functions_bindings object is only referenced in the validate_function_names method, which has a check for None there.

I initially set the value to None to be consistent with the other properties (_require_auth_level), but I think setting it to an empty dict would also work. I'm not sure if that would have the same potential memory leak issue as addressed in https://github.com/Azure/azure-functions-python-library/pull/223 though.

Reply:
Sorry, this comment was meant for the get_functions method.